dashboard/test/mailers/previews/pd_teacher1920_application_mailer_preview.rb

```diff
@@ -0,0 +1,40 @@
+# This can be viewed on non-production environments at /rails/mailers/pd/teacher_application_mailer
+class Pd::Teacher1920ApplicationMailerPreview < ActionMailer::Preview
+  include FactoryGirl::Syntax::Methods
+
+  def confirmation__with_partner
```

Comment:
is the double underscore intentional?

Reply:
Yes, it's a convention I started using back in the workshop mail preview (see https://github.com/code-dot-org/code-dot-org/blob/26d211d9627aca201d25ec55d7ee4c767fb0a261/dashboard/test/mailers/previews/pd_workshop_mailer_preview.rb#L17), to clearly differentiate the mailer action from modifiers on the context.

The name break down like this: `mailer_action_name(__data_modifier)`
So in this case, the mailer action is [confirmation](https://github.com/code-dot-org/code-dot-org/pull/24950/files#diff-75a50d6eac29f50edcd70d5cc7a5e5bcR5), and it has 2 previews available: one `with_partner` and one `without_partner`. Those each describe the different contexts for the same mailer action.